src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,198 @@
+:::
+
+## Set up the extension
+
+### 1. Download and install the extension
+
+The extension is currently supported in the following browsers. We plan to support more browsers in the future. 
+
+* <LinkOut href="https://chromewebstore.google.com/detail/web-monetization-extensio/oiabcfomehhigdepbbclppomkhlknpii">Google Chrome and Chromium-based browsers</LinkOut>
+* <LinkOut href="https://addons.mozilla.org/en-US/firefox/addon/web-monetization-extension/">Mozilla Firefox</LinkOut>
+* <LinkOut href="https://microsoftedge.microsoft.com/addons/detail/web-monetization-extensio/imjgemgmeoioefpmfefmffbboogighjl">Microsoft Edge</LinkOut>
+
+:::note[Permissions]
+The extension will ask for permission to access your browser tabs and your data for all websites. The extension requires these permissions to check each page you visit for the Web Monetization [`<link>`](/docs/references/html-link-rel-monetization) element.
+:::
+
+### 2. Sign up with a compatible digital wallet
+
+Sign up with a Web Monetization-compatible [wallet provider](/docs/resources/op-wallets).
+
+After you've created your account, you'll be provided with a type of URL called a *wallet address*. You need this address in the next step. 
+
+```html title="Example wallet address"
+https://wallet.example.com/alice
+```
+
+### 3. Configure the extension
+
+<Steps>
+1. Open the extension and enter your wallet address in the given field.
+    * Example wallet address - `https://wallet.example.com/alice`
+    <img src="" alt="Extension setup screen with wallet address and amount fields, renew monthly toggle, and connect button" style="max-width:300px" class="img-outline"/>
+2. Enter the amount to make available to spend. 
+3. If you want the amount to renew each month, select the **Renew monthly** toggle.
+4. Click **Connect**.
+5. Click **Agree** to allow the extension to automatically connect with your wallet provider and add a security key to your wallet account.<br /><img src="" alt="Consent screen to allow the extension to connect with the wallet provider with agree and decline buttons" style="max-width:300px" class="img-outline"/><br />If your wallet provider's website opened in your browser, proceed to the next step. If you received a connection error, review the instructions for [resolving the error](#resolve-a-wallet-connection-error).
+6. Sign in to your wallet account if you haven't already, then confirm the connection and budget amount. Below is an example screen.<br /><img src="" alt="Example budget confirmation screen with accept and decline buttons" style="max-width:400px" class="img-outline" />When the connection is successful, your browser will show a success screen.<br /><img src="" alt="Web page displaying a connection successful screen" style="max-width:400px" class="img-outline"/>
+</Steps> 
+
+## Adjust your budget 
+
+<Steps>
+1. Click the cog in the upper-right of the extension.
```

Comment:
Is "cog" a general word? Should we say "Settings icon" instead given we're showing what icon in image next?

Reply:
Will change